### ISSUE TITLE
Windows on ARM binaries missing

### ISSUE THREAD

**Issue by itpropro** [neovim/neovim#30744]
### Problem

In the [INSTALL.md](https://github.com/neovim/neovim/blob/master/INSTALL.md) as well as in the software repos the ARM64 binaries for Windows ARM64 are missing.
Only for macos, the arm release is listed:

>Latest [stable release](https://github.com/neovim/neovim/releases/latest)
>   - [macOS x86](https://github.com/neovim/neovim/releases/latest/download/nvim-macos-x86_64.tar.gz)
>   - [macOS arm](https://github.com/neovim/neovim/releases/latest/download/nvim-macos-arm64.tar.gz)
>   - [Linux](https://github.com/neovim/neovim/releases/latest/download/nvim-linux64.tar.gz)
>   - [Windows](https://github.com/neovim/neovim/releases/latest/download/nvim-win64.msi)

It would be great if they could be added, as compiling yourself is not only not well documented for Windows ARM64 and has some problems, it is also not really practical (and for some users not feasable) to self compile for every release or if you just want to install neovim.

### Steps to reproduce

install neovim by any of the documented Windows ways from the docs and not get a arm64 binary on an arm device.

### Expected behavior

getting a native arm binary on Windows when neovim is installed via package manager or manual download.

### Nvim version (nvim -v)

latest

### Vim (not Nvim) behaves the same?

yes

### Operating system/version

Windows 11 24H2

### Terminal name/version

Windows Terminal 1.20

### $TERM environment variable

-

### Installation

tried every method

**Comment by itpropro** [comment]
> Duplicate of [#12521](https://github.com/neovim/neovim/issues/12521)

That issue is closed without a resolution how can it be a duplicate or are the ARM binaries available now?

**Comment by clason** [comment]
Yes, because that issue is "can't fix", and so is this. We are blocked by the (public!) availability of Windows ARM runners on Github (which _are_ available for macOS, which is the sole reason there are dedicated macOS ARM binaries in our official releases). Once they are available, we'll of course make use of them. So there's no need to track anything.

**Comment by etherbound-dev** [comment]
Windows ARM runners are now publicly available!

**Comment by clason** [comment]
[We know](https://github.com/neovim/neovim/pull/33479), but some work is needed to get Neovim to run on them properly.
